src/unity/python/turicreate/toolkits/one_shot_object_detector/one_shot_object_detector.py

```diff
@@ -76,8 +77,16 @@ def __init__(self, state):
     def predict(self, dataset):
         return self.__proxy__['detector'].predict(dataset)
 
-    def evaluate(self, dataset, metric="auto"):
-        return self.__proxy__['detector'].evaluate(dataset, metric)
+    def evaluate(self, dataset, column_name, metric="auto", output_type='dict', iou_threshold=None, 
```

Comment:
column_name => annotations (use the same terminology as OD) = 'annotations' (set the default to annotations). Also, explain this in the API docs